packages/react/src/components/Toggle/Toggle.Skeleton.js

```diff
@@ -10,7 +10,7 @@ import React from 'react';
 import cx from 'classnames';
 import { PrefixContext } from '../../internal/usePrefix';
 
-export default class ToggleSkeleton extends React.Component {
+export class ToggleSkeleton extends React.Component {
```

Comment:
What are your thoughts on if we should follow the same convention as the other files, having the `export { ToggleSkeleton }` at the bottom and just `class ToggleSkeleton ....` up here?